src/main/java/com/helloworld/helloworld/controller/messageController.java

```diff
@@ -0,0 +1,64 @@
+package com.helloworld.helloworld.controller;
+
+import com.helloworld.helloworld.exceptions.NotFoundException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("message")
+public class messageController {

```

Comment:
create  a SERVICE and move all logic to this service. controller should call this service and return data